src/compiler.rs

```diff
@@ -186,32 +183,29 @@ impl AmberCompiler {
     }
 
     pub fn document(&self, block: Block, meta: ParserMetadata, output: String) {
-        let base_path = PathBuf::from(
-            meta.get_path()
-                .expect("Input file must exist in docs generation"),
-        );
+        let base_path = meta.get_path()
+            .map(PathBuf::from)
+            .expect("Input file must exist in docs generation");
```

Comment:
Not sure if this should be thrown as an actual error message (`Message::new_err_msg` instead of panic). I'll have to test this. I know that this snippet was modified by a formatter, but that's something that caught my eye.

```suggestion
            .expect("Input file must exist in docs generation");
```

Reply:
that part I think that is a duplicate of https://github.com/amber-lang/amber/pull/465 so maybe is better to merge that one as first so we can see in this what can be improved

Reply:
Correct me if I'm wrong, but I don't see any difference in the two lines in your code suggestion!

Reply:
I tend to do some minor refactoring on chained statements like this, to remove nested chains.  I think it makes it easier to reason about the logic.

Reply:
Never mind. Tested it and it works all good. The suggestion code was added by mistake. Just removed it.